README.md

```diff
@@ -165,6 +165,21 @@ These variables are used by the service startup scripts in the Docker images, bu
 * `PROTECTED_RECORDINGS_TOKEN_TIMEOUT`: Protected recording link token timeout in minutes. This is the amount of time that the one-time-use link returned in `getRecordings` calls will be valid for. Defaults to 60 minutes (1 hour).
 * `PROTECTED_RECORDINGS_TIMEOUT`: Protected recordings resource access cookie timeout in minutes. This is the amount of time that a user will be granted access to view a recording for after clicking on the one-time-use link. Defaults to 360 minutes (6 hours).
 * `SCALELITE_API_PORT`: Runs the SCALELITE_API in custom port number. Defaults to 3000.
+* `LOCALE`: Change the language that user facing pages displays in (currently supports `en`)
+
+### Customizing Strings
+
+If you'd like to customize the strings on certain error pages returned by Scaltelite (`recording_not_found`), you can do so by duplicating the locale file and changing whatever lines you see fit.
+
+Create the directory `/etc/default/scalelite/locales` and copy over the locale file which you want to customize the strings of. All locales can be found [here](https://github.com/blindsidenetworks/scalelite/tree/master/config/locales).
```

Comment:
Perhaps we can provide a command to copy the locale file out of the docker container instead so they get the right file version? We should probably also note that it might need to be updated when new scalelite versions are release.

Reply:
There's a note on the next line saying that they'll need to keep it updated. As for copying, GL recommends something like `docker run --rm bigbluebutton/greenlight:v2 cat ./sample.env > .env` in its instalation docs. Do you think something similar is good enough?

Reply:
We don't do that anywhere else in our docs though - so it might better to just keep it consistent